lib/sensu-plugins-zfs/version.rb

```diff
@@ -1,8 +1,8 @@
 module SensuPluginsZFS
   module Version
     MAJOR = 2
-    MINOR = 0
-    PATCH = 1
+    MINOR = 1
```

Comment:
please remove all version changes which are done post acceptance. There are several reasons for this:
- PRs are not a FIFO queue, merge order is not guaranteed just because you submitted first
- maintainer may disagree with you on impact of version

By not having the version change in PRs we can reduce the number of times we need to ask someone to rebase.